### DISCUSSION TITLE
how to solve this error:  Error executing lua ...al/share/nvim/lazy/bufferline.nvim/lua/bufferline/ui.lua:119: Segments must be a list stack traceback:

### DISCUSSION THREAD

**Discussion by Create-Peace** [LazyVim/LazyVim#4570] [Q&A]
always occurs error:
E5108: Error executing lua ...al/share/nvim/lazy/bufferline.nvim/lua/bufferline/ui.lua:119: Segments must be a list
stack traceback:
	[C]: in function 'assert'
	...al/share/nvim/lazy/bufferline.nvim/lua/bufferline/ui.lua:119: in function 'get_component_size'
	...al/share/nvim/lazy/bufferline.nvim/lua/bufferline/ui.lua:480: in function <...al/share/nvim/lazy/bufferline.nvim/lua/bufferline/ui.lua:461>
	vim/shared.lua: in function 'get_components'
	...local/share/nvim/lazy/bufferline.nvim/lua/bufferline.lua:56: in function <...local/share/nvim/lazy/bufferline.nvim/lua/bufferline.lua:54>
	[C]: in function 'ui_attach'
	.../.local/share/nvim/lazy/noice.nvim/lua/noice/ui/init.lua:74: in function 'enable'
	...ray/.local/share/nvim/lazy/noice.nvim/lua/noice/init.lua:61: in function 'enable'
	...ray/.local/share/nvim/lazy/noice.nvim/lua/noice/init.lua:23: in function <...ray/.local/share/nvim/lazy/noice.nvim/lua/noice/init.lua:19>
	[C]: in function 'xpcall'
	...local/share/nvim/lazy/noice.nvim/lua/noice/util/call.lua:149: in function 'try'
	...ray/.local/share/nvim/lazy/noice.nvim/lua/noice/init.lua:19: in function <...ray/.local/share/nvim/lazy/noice.nvim/lua/noice/init.lua:18>

**Comment by dpetka2001** [comment]
It would be useful to also share your configuration. What changes you made. Because with LazyVim default installation there's no such error.
